### PR TITLE
Validation: fix duplicate h2 and alt text missing

### PR DESCRIPTION
**Pull request by chrisms86**

### CODE REVIEW

File: aspnet/web-forms/overview/moving-to-aspnet-20/data-bound-controls.md [hunk 1]
```diff
@@ -662,19 +662,21 @@ This exercise assumes that you have SQL Server running locally and that the Nort
 8. Select **Northwind** in the connection dropdown and click Next.
 9. Select **Products** from the **Name** dropdown and check the **ProductID**, **ProductName**, **UnitPrice**, and **UnitsInStock** checkboxes as shown below. 
 
-![](data-bound-controls/_static/image3.jpg)
+![A screenshot of the configure data source - products screen.](data-bound-controls/_static/image3.jpg)
```
Comment (MeeraDi) [inline aspnet/web-forms/overview/moving-to-aspnet-20/data-bound-controls.md:665]: A screenshot of the configure data source products screen. 
Removed the dash for screen readers.

File: aspnet/web-forms/overview/moving-to-aspnet-20/improvements-in-visual-studio-2005.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Web developers have many new options available for project creation in Visual St
 
 Click here to view a video walkthrough of creating Web projects in Visual Studio 2005.
 
-![](improvements-in-visual-studio-2005/_static/image1.png)
+![Screenshot of the video walkthrough of creating Web projects in Visual Studio.](improvements-in-visual-studio-2005/_static/image1.png)
```
Comment (MeeraDi) [inline aspnet/web-forms/overview/moving-to-aspnet-20/improvements-in-visual-studio-2005.md:59]: I don't think this is a screenshot, isn't this more of the icon that they click to start the video?

File: aspnet/web-forms/overview/moving-to-aspnet-20/improvements-in-visual-studio-2005.md [hunk 3]
```diff
@@ -246,7 +246,7 @@ Another great enhancement to Visual Studio and the ASP.NET architecture is in th
 
 Click here to see a video walkthrough of the edit and continue feature in Visual Studio 2005.
 
-![](improvements-in-visual-studio-2005/_static/image2.png)
+![Screenshot of the video walkthrough of the edit and continue feature in Visual Studio.](improvements-in-visual-studio-2005/_static/image2.png)
```
Comment (MeeraDi) [inline aspnet/web-forms/overview/moving-to-aspnet-20/improvements-in-visual-studio-2005.md:249]: Same as above comment.

**Comment by chrisms86** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**